src/scm/github.js

```diff
@@ -312,156 +335,169 @@ class Github {
-      .fail(reject);
-    })
-    .then(response => {
-      const repo = {
-        fullName : response.full_name
-      };
-      context.repo = repo;
-      Object.assign(context.bindRepo, { [context.id] : repo });
-      if (context.bindBranch[context.id]) {
-        delete context.bindBranch[context.id];
-      }
-      chrome.storage.sync.set({ bindRepo: context.bindRepo });
-      return response.full_name;
-    })
-    .catch(err => {
-      throw new Error('Failed to create new repository.');
-    });
+      .then(response => {
+        const repo = {
+          fullName: response.full_name
+        };
+        context.repo = repo;
+        Object.assign(context.bindRepo, {
+          [context.id]: repo
+        });
+        if (context.bindBranch[context.id]) {
+          delete context.bindBranch[context.id];
+        }
+        chrome.storage.sync.set({
+          bindRepo: context.bindRepo
+        });
+        return response.full_name;
+      })
+      .catch(err => {
+        throw new Error('Failed to create new repository.');
+      });
   }
 
   createGist() {
     const desc = $('#new-gist-name').val();
     const isPublic = $('#new-gist-public').val() !== 'secret';
     const payload = {
-      'description' : desc,
+      'description': desc,
       'public': isPublic,
       'files': {
-        'init_by_gas_hub.html' : {
+        'init_by_gas_hub.html': {
           'content': 'init by gas-hub, just delete this file.'
         }
       }
     };
 
     return new Promise((resolve, reject) => {
-      $.ajax({
-        url: `${this.baseUrl}/gists`,
-        headers: {
-          'Authorization': `token ${this.accessToken}`
-        },
-        method: 'POST',
-        crossDomain: true,
-        dataType: 'json',
-        contentType: 'application/json',
-        data: JSON.stringify(payload)
+        $.ajax({
+            url: `${this.baseUrl}/gists`,
+            headers: {
+              'Authorization': `token ${this.accessToken}`
+            },
+            method: 'POST',
+            crossDomain: true,
+            dataType: 'json',
+            contentType: 'application/json',
+            data: JSON.stringify(payload)
+          })
+          .then(resolve)
+          .fail(reject);
       })
-      .then(resolve)
-      .fail(reject);
-    })
-    .then(response => {
-      const gist = response.id;
-      context.branch = gist;
-      Object.assign(context.bindBranch, { [context.id] : gist });
-      chrome.storage.sync.set({ bindBranch: context.bindBranch });
-      return response;
-    })
-    .catch(err => {
-      throw new Error('Failed to create new gist.');
-    });
+      .then(response => {
+        const gist = response.id;
+        context.branch = gist;
+        Object.assign(context.bindBranch, {
+          [context.id]: gist
+        });
+        chrome.storage.sync.set({
+          bindBranch: context.bindBranch
+        });
+        return response;
+      })
+      .catch(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/github.js#L420"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>